app/volunteer-remove-confirm/[[...volunteer-remove-confirm]]/page.tsx

```diff
@@ -1 +1,39 @@
-// Write your code here
\ No newline at end of file
+// imports
+import { ButtonExit, ButtonSubmit, ButtonEdit } from "../../components/SurveyButtons";
+import UpdateInventoryBanner from "../../components/UpdateInventoryBanner";
+
+const RemoveConfirmPage: React.FC = () => {
+    
+    return (
+        <div>
+            {/* Imported Banner */}
+            <div>
+                <UpdateInventoryBanner/>
+            </div>
+            <div>
+                {/* Top Buttons */}
+                <div className="flex p-20 crimson-regular text-2xl content-center justify-between">
+                    <ButtonEdit/>
+                    <ButtonExit/>
+                </div>
+
+                {/* Central text */}
+                <div className="text-black crimson-bold flex text-4xl content-center justify-center text-center">
+                    This action will:
+                </div>
+
+                {/* Placeholder text */}
+                <div className="text-gray crimson-regular pt-10 flex text-4xl content-center justify-center text-center">
+                    Remove [quantity] [units] of [itemName].
+                </div>
+                
+                {/* Bottom Buttons */}
+                <div className="flex pt-[250px] crimson-regular text-2xl justify-center">
```

Comment:
<img width="1440" alt="Screenshot 2024-11-15 at 7 08 52 PM" src="https://github.com/user-attachments/assets/9b46da1d-a3e7-4ef9-9d2f-e97ab139bb65">

^^ Same nitpick comment as the `volunteer-add-confirm` comment (about moving the submit button higher)